.github/workflows/push.yml

```diff
@@ -52,15 +63,18 @@ jobs:
       image_name: ${{ env.GHA_DOCKER_PUSH_IMAGE_NAME }}
```

Comment:
This is still vulnerable. 

Change from `image_name: ${{ env.GHA_DOCKER_PUSH_IMAGE_NAME }}` to `image_name: ${GHA_DOCKER_PUSH_IMAGE_NAME}`